proto/notifications/v1/service.proto

```
@@ -0,0 +1,81 @@
+message SubscribeResponse {}
+
+// Unsubscribe from a list of topics
+message UnsubscribeRequest {
+    string installation_id = 1;
+    repeated string topics = 2;
+}
+
+// Response to UnsubscribeRequest
+message UnsubscribeResponse {}
+
+service Notifications {
```

Buf (1.54.0)

Service name "Notifications" should be suffixed with "Service".

(SERVICE_SUFFIX)